shell/platform/fuchsia/flutter/platform_view.cc

```diff
@@ -1062,6 +1106,12 @@ bool PlatformView::HandleFlutterPlatformViewsChannelPlatformMessage(
     on_destroy_view_callback_(view_id_raw, std::move(on_view_unbound));
   } else if (method.rfind("View.focus", 0) == 0) {
     return focus_delegate_->HandlePlatformMessage(root, message->response());
+  } else if (method == "HostView.getCurrentFocusState") {
```

Comment:
What's going on with these changes?

Reply:
looks like @jaeheon needs to rebase.

Reply:
Yeah, something goofy happened. I see Darren's deletion CL in my git history, so I'll manually clean this up.